### DISCUSSION TITLE
Modal set focus on open

### DISCUSSION THREAD

**Discussion by dominikgeimer** [wireui/wireui#388] [Q&A]
Hi,

I have a Modal with in input inside of it and I am opening the Modal with Livewire `wire:model.defer="openSearchModal"`.

Is there a way with the Modal Events

```
<x-modal
    ...
    x-on:open='...'
    x-on:close='...'>
 
</x-modal>
```

To set the focus to the input element?

**Comment by ph7jack** [comment]
@dominikgeimer I'll create a feature to do it. Maybe the alpine x-trap can work
